.travis.yml

```diff
@@ -10,7 +10,7 @@ rvm:
   - ruby-head
   - rbx-2
 
-install: bundle install --path=vendor/bundle --retry=3 --jobs=3
+install: bundle install --path=vendor/bundle --retry=3 --jobs=3 --without bench
```

Comment:
Seems like travis is not working because it uses an old version of bundler that does not handle the `--without` parameter as expected. We can either follow https://github.com/travis-ci/travis-ci/issues/3531#issuecomment-88311203 or maybe use `--without development bench`?